addons/event/data/email_template_data.xml

```diff
@@ -55,6 +55,29 @@ from ${object.event_id.date_begin_located} to ${object.event_id.date_end_located
 <p>If you have any questions, please let us know.</p>
 <p>Best regards,</p>]]></field>
         </record>
-
+        <record id="event_change_notification" model="mail.template">
+            <field name="name">Event: schedule change</field>
+            <field name="model_id" ref="event.model_event_registration"/>
+            <field name="email_from">${(object.event_id.organizer_id.email or object.event_id.user_id.email or '')|safe}</field>
+            <field name="email_to" >${object.email|safe}</field>
```

Comment:
email field is not required. so what happen if there is not value in email for registration. ?